rotkehlchen/exchanges/manager.py

```diff
@@ -231,7 +245,11 @@ def setup_exchange(
             api_secret=api_secret,
             passphrase=passphrase,
         )
-        exchange = self.initialize_exchange(
+        kwargs.update({
```

Comment:
why do you need to do this? it should work without this. we do something similar for binance.